jdi-light-html/src/main/java/com/epam/jdi/light/ui/html/elements/common/FileInput.java

```diff
@@ -34,4 +43,41 @@ public String getValue() {
     // endregion
     @Override
     public TextAssert is() { return new TextAssert().set(this); }
+
+    @JDIAction("Upload file '{0}' for '{name}'")
+    public void uploadFileRobot(String path, long mSec)  {
+
+        Actions builder = new Actions(uiElement.driver());
+        builder.moveToElement(uiElement.getWebElement()).click().build().perform();
```

Comment:
looks like we should have jdi own methods to do it instead of Actions

Reply:
ok